apps/cranio-provider/tests/index.test.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test } from "vitest";
+import {
+  asKeyValuePairs,
+  sum,
+  sumObjectValues,
+  uniqueValues,
+} from "../src/utils/";
+
+const data = [
+  { group: "A", value: 32, count: 4 },
+  { group: "B", value: 22, count: 10 },
+  { group: "C", value: 19, count: 2 },
+];
+
+const dataObject = asKeyValuePairs(data, "group", "value");
+
+test("data is transformed into an object of key-value pairs", () => {
```

Comment:
Wrap each different function you test in a 'describe("functionName",()=>{...tests...});'